modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/configuration/DistributionZoneConfigurationSchema.java

```diff
@@ -67,4 +67,8 @@ public class DistributionZoneConfigurationSchema {
     @Range(min = 0)
     @Value(hasDefault = true)
     public int dataNodesAutoAdjustScaleDown = INFINITE_TIMER_VALUE;
+
+    @ValidFilter
+    @Value(hasDefault = true)
+    public String filter = "$..*";
```

Comment:
And what if user will try to change the filter from some to "anything"? Do you expect them to specify "$..*"?